### ISSUE TITLE
Tags badge in blog post not rendering

### ISSUE THREAD

**Issue by grandesballo** [puresyntax71/hugo-theme-chunky-poster#101]
I'm trying to make tags badge appear in blog posts.
I se that in single.html the code is present, but tags don't appear.

I tried to add tags in the header copying them from a page of your exampleSite, but the site then doesn't render and gives an error because the heaer is formatted
`tags = ["value","value2"]`
but Hugo is expecting
`tags: ["value","value2"]`

is that the cause of the issue?

**Comment by puresyntax71** [comment]
Yeah, this would depend on the [front-matter](https://gohugo.io/content-management/front-matter/) you use i.e. using `+++` means it's TOML which would be `tags = ["value","value2"]` whilst `---` means it's YAML which would be `tags: ["value","value2"]`.
